README.rst

```diff
@@ -5,18 +5,23 @@ Client interface for Scrapinghub API
 .. image:: https://secure.travis-ci.org/scrapinghub/python-scrapinghub.png?branch=master
    :target: http://travis-ci.org/scrapinghub/python-scrapinghub
 
+
+The ``scrapinghub`` is a Python library for communicating with the `Scrapinghub API`_.
+
+
+.. contents:: :depth: 1
+
+
 Requirements
 ============
 
 * Python 2.6 or above
 * `Requests`_ library
```

Comment:
You're right, though I'd leave Python itself in requirements and that's all.
Added Installation section 👍